lib/api/controllers/remoteActions/managePlugins.js

```diff
@@ -9,21 +9,22 @@ var
   lockfile = require('proper-lockfile'),
   _ = require('lodash'),
   clc = require('cli-color'),
-  ok = clc.green.bold,
   defaultConfig = require('rc')('kuzzle'),
-  DatabaseService = require('../../../services/internalEngine');
-
-var
-  clcError = clc.red,
+  DatabaseService = require('../../../services/internalEngine'),
+  clcOk = clc.green.bold,
   clcNotice = clc.cyan,
-  clcOk = clc.green.bold;
+  clcError = clc.red;
 
 module.exports = function pluginsManager(plugin, options) {
   var
     kuzzleConfiguration = require('../../../config')(defaultConfig),
     dbService = new DatabaseService({config: kuzzleConfiguration}),
     deferred = q.defer();
-
+  
+  if (options.parent && options.parent.noColors) {
```

Comment:
These functions are used outside of module.exports, where the options variable does not exist.
It was much easier, notably regarding the tests we have, to keep the default behavior and override the methods only when asked.